src/apps/acb2wavs/acb2wavs.cpp

```diff
@@ -48,9 +48,12 @@ int main(int argc, const char *argv[]) {
 
 void PrintHelp() {
     cout << "Usage:\n" << endl;
-    cout << "acb2wavs <acb file> [-a <key1> -b <key2>]" << endl;
+    cout << "acb2wavs <acb file> [-a <key1> -b <key2> -n]" << endl;
+    cout << "\t-n\tUse cue names for output waveforms]" << endl;
 }
 
+static char useCueName = 0;
```

Comment:
Don't use global variables. Use parameters and arguments. Also there is `bool_t`.